portal-articles/WebsitesExtension/WebApps/Overview.md

```diff
@@ -11,13 +11,15 @@ ms.date: 04/27/2017
 ms.author: sewatson
```

Comment:
`ms.date` needs to be set to current date

Reply:
Maybe @sewatson can fix that when he pushes to live.